src/Assert.php

```diff
@@ -1209,6 +1235,8 @@ public static function alnum($value, $message = '')
      */
     public static function lower($value, $message = '')
     {
+        static::string($value, $message);
```

Comment:
The ctype based methods return false when not provided with a string, so on that front it is not a BC break. (objects that can be cast to string won't be cast either, it will just be false).

The only problem with them is that ints between -128 and 255 get converted to their corresponding char in the ASCII table. Which is _probably_ not intended behaviour for the users.